examples/custom-server/server.js

```diff
@@ -1,5 +1,5 @@
 const { createServer } = require('http')
-const { parse } = require('url')
+const { URL } = require('url')
```

Comment:
Since we support Node.js 10+ and the `docs` under the `URL` class has the following message:

> The class is now available on the global object.

I think we can remove the import 🤔 

```suggestion
```